### DISCUSSION TITLE
On Eurelian model, how to enable the GENTOP method?

### DISCUSSION THREAD

**Discussion by MayconSilva7** [ansys/pyfluent#1884] [General]
Hello, 

I would like to set the GENTOP method on regime transition modeling, but it is not available in the documentation. As you see in the following image.

![image](https://github.com/ansys/pyfluent/assets/122628753/e7567409-fa37-41fa-b483-0bec03927c49)

How do I enable this method?

Thank you very much

**Comment by MayconSilva7** [comment]
Hello. Is there anyone who can help me with this, please? 

Is the GENTOP method for multiphase flow available? 

Thank you very much.

**Comment by raph-luc** [comment]
Hi @MayconSilva7, after launching Fluent using PyFluent and loading your mesh, try this:
```
solver.tui.define.models.multiphase.model("eulerian")
solver.tui.define.models.multiphase.regime_transition_modeling("no","yes","3")
```

Tested this using Fluent 2023 R2.

The first command enables the Eulerian inhomogeneous model which is required for GENTOP, and in the second command "no" is for AIAD, "yes" for GENTOP, and "3" for the number of  Eulerian phases.

Currently, you can figure out what commands are required by looking at the Fluent documentation, and trying the TUI commands inside Fluent itself.

**Reply by MayconSilva7** [reply]
Thank you very much. 

I have been looking at the documentation and the definitions of the class and functions. But sometimes I could not find how to set a command. 

Thank you again!
